usagov/usagov-2021
# usagov-1319-benefits-finder-v0-1-1-beta-1: Changes for the new release

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1319

## Description

Integrate Benefits Finder v0.1.1.beta.1 Release

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


You can follow the "Detailed Testing steps" in each of these Pull Request(s) to confirm integration. (I summarize them below on how I validated) 
[#739](https://github.com/GSA/px-benefit-finder/pull/739) [#759](https://github.com/GSA/px-benefit-finder/pull/759) [#760](https://github.com/GSA/px-benefit-finder/pull/760) [#716](https://github.com/GSA/px-benefit-finder/pull/716) [#644](https://github.com/GSA/px-benefit-finder/pull/644) [#715](https://github.com/GSA/px-benefit-finder/pull/715) 
(Note: the code scans in the second link are failing, but not because of this change. They're catching a recent vulnerability in a dependency.)

### Requires New Config

### Requires New Content
- [x] Yes (if you don’t have benefits content already 

### Validation Steps
[#739](https://github.com/GSA/px-benefit-finder/pull/739) Shows an alert warning when you press the browser back button that changes may be lost 
[#759](https://github.com/GSA/px-benefit-finder/pull/759) Should be able to tab to cards in the more benefits section
[#760](https://github.com/GSA/px-benefit-finder/pull/760) Focuses the alert element on error when present 
[#716](https://github.com/GSA/px-benefit-finder/pull/716) this updates our fetch method to handle a proxy request during local dev to the hosted s3 in cloud
[#644](https://github.com/GSA/px-benefit-finder/pull/644) updates node to v20+ in the .nvmrc and lock file 
[#715](https://github.com/GSA/px-benefit-finder/pull/715) This updates our parseDate utility function to include the calculation for time offset due to the limited object values passed on new Date creation
	•	Navigate through form taking note of dates entered
	•	Select review selections
	•	Ensure that the dates selected match those rendered in the review listing

[#610](https://github.com/GSA/px-benefit-finder/pull/610)
changes  "Antes de empeza" to "Antes de empezar" in spanish intro text 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
